From b14e2812307c8107e7f88cf1f701fcace0d03d03 Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Mon, 31 Aug 2020 17:32:50 +0800
Subject: [PATCH] media: i2c: imx378: select V4L2_FWNODE

Building imx378 fails now unless V4L2_FWNODE is selected:

drivers/media/i2c/imx378.o: In function `imx378_probe':
imx378.c:(.text+0x1074): undefined reference to `v4l2_async_register_subdev_sensor_common'

Fixes: 797d705c9037 ("media: i2c: add imx378 sensor driver")
Change-Id: Ie89333159f76d435dc253d9b025bc83c949066ec
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 drivers/media/i2c/Kconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/media/i2c/Kconfig b/drivers/media/i2c/Kconfig
index 6fc0114e514e..a5163c63664b 100644
--- a/drivers/media/i2c/Kconfig
+++ b/drivers/media/i2c/Kconfig
@@ -749,6 +749,7 @@ config VIDEO_IMX378
 	tristate "Sony IMX378 sensor support"
 	depends on I2C && VIDEO_V4L2 && VIDEO_V4L2_SUBDEV_API
 	depends on MEDIA_CAMERA_SUPPORT
+	select V4L2_FWNODE
 	help
 	  This is a Video4Linux2 sensor driver for the Sony
 	  IMX378 camera.
-- 
2.35.3

